### PR TITLE
Code sample - SpringBoot Hikari to UCP migration

### PR DESCRIPTION
**Pull request by juarezjuniorgithub**
Signed-off-by: Juarez Barbosa juarez.barbosa@oracle.com
@Kuassim @jeandelavarene please review and approve. Thanks!

### CODE REVIEW

File: java/springboot-hikari-ucp/pom.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>23.3.0.23.09</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ucp11</artifactId>
+			<version>23.3.0.23.09</version>
```
Comment (jeandelavarene) [inline java/springboot-hikari-ucp/pom.xml:54]: Please use 23.4

Comment (juarezjuniorgithub) [reply java/springboot-hikari-ucp/pom.xml]: Done

Comment (jeandelavarene) [reply java/springboot-hikari-ucp/pom.xml]: Please use [23.4.0.24.05](https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc11/23.4.0.24.05)

Comment (juarezjuniorgithub) [reply java/springboot-hikari-ucp/pom.xml]: Hi Jean, it's already using it. You're looking at a past commit, I followed all your requests. To confirm, look at commit 75a6f82896adb503af24d42cf1e149cb96689c02

**Review by Kuassim — APPROVED** [review]

File: java/springboot-hikari-ucp/pom.xml (hunk 1 above)
Comment (juarezjuniorgithub) [reply java/springboot-hikari-ucp/pom.xml]: @Kuassim please review, thanks

**Review by Kuassim — APPROVED** [review]

**Pull request closed without merging**